utils/clean_up_repo.py

```diff
@@ -0,0 +1,33 @@
+"""
+The Qxf2 automation repository ships with example tests.
+Run this file to delete all the example files and start fresh with your example.
+After this script runs, you will need to edit a few files to configure them to suit your own repository
+Usage: python clean_up_repo.py
+"""
+import os,sys
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from conf import clean_up_repo_conf as conf
+
+def delete_file(filename):
+    # The method will delete a particular file
+    if os.path.exists(filename):
+        os.remove(filename)
+        print(f'{filename} deleted')
+
+def delete_files_in_dir(directory, files):
+    # The method will delete files in a particular directory
+    for filename in files:
+        delete_file(os.path.join(directory,filename))
+
+def delete_files_used_in_example():
+    # The method will delete files mentioned in the blog
+    # Blog URL: https://qxf2.com/blog/how-to-start-using-the-qxf2-framework-with-a-new-project/
+    for every_dir_list in conf.dir_list:
+            for every_file_list in conf.file_list:
```

Comment:
This logic is incorrect. You will loop through each file multiple times - as many times as there are directories. Either look up zip(dir_list, file_list) or just make conf to have list like this:

files_to_delete = [(CONF_DIR, CONF_FILES_DELETE),(ENDPOINTS_DIR, ENDPOINTS_FILES_DELETE), etc, etc.]

Reply:
Used zip now:
`for every_dir_list, every_file_list in zip(conf.dir_list,conf.file_list):
        delete_files_in_dir(every_dir_list, every_file_list)`